lib/models/authors.js

```diff
@@ -0,0 +1,62 @@
+const pool = require('../utils/pool');
+
+class Author {
+  id;
+  name;
+  dob;
+  pob;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.dob = row.dob;
+    this.pob = row.pob;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query(`
+    SELECT * FROM authors`);
+    return rows.map((authorRow) => new Author(authorRow));
+  }
+}
+
+class AuthorDeets {
```

Comment:
don't do this — you want one model for your Author, and that model will have multiple methods that you write (getAll, getById). check out all the code you had to duplicate in this file, very WET (write everything twice) not very DRY (don't repeat yourself)

basically, you want one model per table / controller